docs/events/timeline-ui-overview.asciidoc

```diff
@@ -2,84 +2,72 @@
 = Investigate events in Timeline
 
 Use Timeline as your workspace for investigations and threat hunting.
-Data from multiple indices can be added to a Timeline, which enables
-investigating complex threats.
+You can add data from multiple indices to a Timeline to facilitate advanced investigations.
```

Comment:
```suggestion
You can add alerts from multiple indices to a Timeline to facilitate advanced investigations.
```

Reply:
Alerts might be a bit more precise in this sentence and it's the actual action that users are doing.